{{ cookiecutter.app_name }}/apt.txt

```diff
@@ -0,0 +1,4 @@
+libpq-dev
+python-pip
+python-dev
+python3-dev
```

Comment:
The tox config contains `envlist = py27,py33,py34`, but this will never result in Python 3.3 being installed. Should `py33` be removed from that list?